### PR TITLE
fix(infra-logs): add improvement

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

[> Link to the JIRA ticket
](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-454)

- Add some UI improvements for infra logs errors

---

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [ ] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.  
Click on the link below to follow its deployment and use it.  
:point_right: [[PR] staging - fix(infra-logs): add improvement - 2022-07-08T09:55:00Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/4b539933-db86-4ce4-ba14-56fa46f1cbad/applications)

File: libs/shared/ui/src/lib/components/layouts/layout-logs/tabs-logs/tabs-logs.tsx [hunk 1]
```diff
@@ -77,18 +73,27 @@ export function TabsLogs(props: TabsLogsProps) {
...
-                        <div className="cursor-pointer" onClick={() => copyToClipboard(errors[0].error)}>
-                          <Icon name="icon-solid-copy" className="text-xs text-text-300 hover:text-text-100" />
-                        </div>
-                      </Tooltip>
+                      <div className="flex text-xs">
+                        <p
+                          onClick={() => scrollToError()}
+                          className="transition-colors cursor-pointer text-error-500 hover:text-error-600 font-bold mr-2.5"
+                        >
+                          <span className="text-xxs relative -top-[1px] mr-1">{errors[errors.length - 1].index}</span>
```
Comment (WilliamTraoreee) [inline libs/shared/ui/src/lib/components/layouts/layout-logs/tabs-logs/tabs-logs.tsx:89]: You can change `-top-[1px]` with `-top-px`

**Review by WilliamTraoreee — APPROVED** [review]
Good job @RemiBonnet ! Just a tiny comment but overall it's good :)

**Pull request merged**